python/test/pipeline_test.py

```diff
@@ -28,6 +28,7 @@
 import unittest
 import urllib
 import urlparse
+import pytz
```

Comment:
I don't think it is a standard library in Python (at least not 2.7) and I checked around and it does not seem to get bundled with AE SDK/runtime.